### DISCUSSION TITLE
续费专业版问题

### DISCUSSION THREAD

**Discussion by musan318** [getlantern/lantern-forum-cn#699] [General]
我的专业版到期时间是8月2日，但7月5日就无法使用专业版，自动转为了免费版，

我于7月5日又续费了专业版两年，但续费后主设备的蓝灯一直闪退，无法使用；副设备则一直还是无法自动同步为专业版，均无法使用，最近几个月的使用体验极差

**Comment by MaxMKMax** [comment]
@musan318 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!
